dev.js

```diff
@@ -17,13 +16,20 @@ const {
     externalizeClasses: {
       type: 'boolean',
     },
+    omitComponentClasses: {
+      type: 'boolean',
+    },
     usePixels: {
       type: 'boolean',
     },
   },
 });
-
-const uno = createGenerator({ presets: [presetWarp( { ...options, development: true } )] });
+const asyncPreset = await presetWarp({
+  ...options,
+  omitComponentClasses: true,
```

Comment:
Do we want to omit component classes when testing what UnoCSS generates in the terminal (`pnpm dev -c someclass`)? Then we won't get any output for classes used in components, which might be confusing when we're checking why some classes allegedly don't work with our preset. Wdyt?